db/models.py

```diff
@@ -1 +1,38 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        related_name="skill_set",
+        on_delete=models.CASCADE
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
```

Comment:
In the Guild model, the description field is set to allow null values. In Django, it's generally recommended to use blank=True instead of null=True for string-based fields, as it better aligns with Django's conventions for handling empty strings.